content/induction/sprints/1/prep/index.md

```diff
@@ -0,0 +1,358 @@
+
+{{<note type="question" title="❓ Question: ">}}
+
+How can we get a copy of an existing Github repository on our local machine?
+
+{{</note>}}
+
+In other words, we need to get a local copy of the repository on Github.
+
+{{<note type="note" title="cloning">}}
+
+We call a local copy of a Github repository a **clone**.
+
+The process of copying a remote repository on to a local machine is known as **cloning**.
+
+A repository on Github is said to be **remote**.
+
+A repository on our own computer is said to be **local**.
+
+{{</note>}}
+
+{{<note type="exercise" title="Exercise 5.1 - 👣 Follow the steps">}}
+
+###
+
+> 🎯 Goal: Clone a remote repository to a local machine
+
+You'll need to use the information below to achieve this goal:
+
+You'll need to clone the following repository 👉 https://github.com/CodeYourFuture/education-blog
+
+Follow through the steps in this video 👉 https://www.youtube.com/watch?v=ILJ4dfOL7zs ( Remember to use the url for the education-blog repo when you are cloning )
+
+{{</note>}}
+
+## 📘 Viewing the files
+
+Once you've got a local copy of a codebase on your local machine you can start to view the files and folders in that codebase. You can use a code editor like VSCode.
+
+VSCode is an application that enables developers to view and edit files on their local machine.
+
+{{<note type="exercise" title="Exercise 5.1">}}
+
+### Explore VSCode
+
+Figure out how to open the cloned repository on your local machine in VSCode.
+
+Explore the repository in VSCode and use the code editor to look at the various files and folders.
+
+Try opening the Integrated Terminal in your VSCode window
+
+{{</note>}}
+
+## 🌳 Branching
+
+We can check the commits on the remote repository as before:
+
+![commit-history](commit-history.png)
+
+On the left page of the page, we see additional information:
+
+![main-branch-highlighted](main-branch-highlighted.png)
+
+> So what is **`main`**?
+
+`main` is a **branch**.
+
+Commits form a sequence that look like this:
+
+```mermaid
+gitGraph
+       commit
+       commit
+       commit
+```
+
+The **main branch** is a sequence of commits. It represents a particular line of development in a project.
+
+{{<note type="definition" title="definition: Branch">}}
+
+A **branch** is a sequence of commits in a project.
+
+{{</note>}}
+
+Often we want to start work on a particular change _without_ altering the history in a particular branch.
+
+Instead we can create a new branch to start producing our own work.
+
+We can start to create independent branches like this:
+
+```mermaid
+gitGraph
+        commit
+        commit
+        branch "week-1-coursework"
+        commit
+        commit
+        commit
+```
+
+In the diagram above, we can continue to commit on the "week-1-coursework" branch without altering the history of the `main` branch.
+
+{{<note type="exercise" title="exercise 6.1">}}
+
+### Creating a local branch
```

Comment:
Cool, I did wonder about this - I think I'll link them to a video reference they can use to try and figure this out. If they don't quite get it in then it will be covered in the Git workshop on Saturday.